.github/workflows/tests.yaml

```diff
@@ -7,7 +7,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.6, 3.8, 3.9]
+        python-version: [3.6, 3.7, 3.8, 3.9]
```

Comment:
Reminder to update the README's "python 3.5 or greater" to be "python 3.6 or greater".